.github/workflows/phpunit.yml

```
@@ -8,12 +8,9 @@ on:
 
 jobs:
   PHPUnit:
-    uses: discoverygarden/phpunit-action/.github/workflows/phpunit.yml@v1
+    #uses: discoverygarden/phpunit-action/.github/workflows/phpunit.yml@v1
+    uses: discoverygarden/phpunit-action/.github/workflows/phpunit.yml@adam-vessey-patch-1
```

To be moved back prior to merge, after merge of https://github.com/discoverygarden/phpunit-action/pull/24

```suggestion
    uses: discoverygarden/phpunit-action/.github/workflows/phpunit.yml@v1
```